tarodb/sqlc/queries/assets.sql

```diff
@@ -238,6 +238,31 @@ JOIN genesis_info_view
 JOIN key_group_info_view
     ON assets.genesis_id = key_group_info_view.gen_asset_id;
 
+-- name: FetchGroupByGroupKey :one
+SELECT 
+    key_group_info_view.gen_asset_id AS gen_asset_id,
+    key_group_info_view.raw_key AS raw_key,
+    key_group_info_view.key_index AS key_index,
+    key_group_info_view.key_family AS key_family
+FROM key_group_info_view
+WHERE (
+    key_group_info_view.tweaked_group_key = @group_key
+)
+-- Sort and limit to return the genesis ID for initial genesis of the group.
+ORDER BY key_group_info_view.sig_id
+LIMIT 1;
+
+-- name: FetchGroupByGenesis :one
+SELECT
+    key_group_info_view.tweaked_group_key AS tweaked_group_key,
+    key_group_info_view.raw_key AS raw_key,
+    key_group_info_view.key_index AS key_index,
+    key_group_info_view.key_family AS key_family
+FROM key_group_info_view
+WHERE (
+    key_group_info_view.gen_asset_id = @genesis_id
```

Comment:
Cool, this works as the `gen_asset_id` field must be unique for the `asset_group_sigs` table, which this view uses as the primary table in the join.